packages/react/src/generators/cypress-component-configuration/cypress-component-configuration.ts

```diff
@@ -6,7 +6,7 @@ import {
 } from '@nx/devkit';
 import { nxVersion } from '../../utils/versions';
 import { addFiles } from './lib/add-files';
-import { FoundTarget, addCTTargetWithBuildTarget } from '../../utils/ct-utils';
+import { addCTTargetWithBuildTarget, FoundTarget } from '../../utils/ct-utils';
```

Comment:
the `FoundTarget` interface must have been defined twice. on in react and one in cypress. probably just consolidate to the one in cypress instead?

Reply:
yep!